Makefile

```
@@ -0,0 +1,14 @@
+it-now: clean install test
```

Would you mind renaming this to "default", since that's what the first rule is in a Makefile?
```suggestion
default: clean install test
```